cmd/steal.go

```diff
@@ -65,6 +66,7 @@ func NewStealCmd() *cobra.Command {
 	persistentFlags.StringVarP(&opts.configPath, "config", "c", config.DefaultConfigFileName, "Path to config file")
 	persistentFlags.StringVarP(&opts.from, "from", "f", "mysql://root:root@tcp(localhost:3306)/klepto", "Database dsn to steal from")
 	persistentFlags.StringVarP(&opts.to, "to", "t", "os://stdout/", "Database to output to (default writes to stdOut)")
+	persistentFlags.BoolVar(&opts.toRDS, "to-rds", true, "If the output server is an AWS RDS server")
```

Comment:
Setting the flag to `true` as default value changes default command behaviour, if I understand this correctly, and we should avoid this as some people may use the app in the pipelines.

```suggestion
	persistentFlags.BoolVar(&opts.toRDS, "to-rds", false, "If the output server is an AWS RDS server")
```